Sources/Sentry/Public/SentrySpanProtocol.h

```diff
@@ -27,7 +27,12 @@ NS_SWIFT_NAME(Span)
 /**
  * An arbitrary mapping of additional metadata of the span.
  */
-@property (nullable, readonly) NSDictionary<NSString *, id> *data;
+@property (readonly) NSDictionary<NSString *, id> *data;
```

Comment:
`h`: Unfortunately removing `nullable` is a breaking change. We can't do that.

Reply:
I created an issue to fix this in the next major. https://github.com/getsentry/sentry-cocoa/issues/1257

Reply:
Got carried away to make data implementation the same as tags.
Rolling it back.